examples/language-modeling/README.md

```diff
@@ -453,8 +453,43 @@ LOWER_LIST=ops_bf16.txt python3 ../gaudi_spawn.py \
     --adam_epsilon 1e-08 \
     --do_eval \
     --low_cpu_mem_usage True
-```
 
+- Multi-card finetuning of Falcon-180B:
+```bash
+DEEPSPEED_HPU_ZERO3_SYNC_MARK_STEP_REQUIRED=1 LOWER_LIST=ops_bf16.txt python3 ../gaudi_spawn.py \
+    --world_size 8 --use_deepspeed run_lora_clm.py \
+    --model_name_or_path tiiuae/falcon-180B \
+    --dataset_name timdettmers/openassistant-guanaco \
+    --bf16 True \
+    --output_dir ./model_lora_falcon_ddp \
+    --num_train_epochs 3 \
+    --per_device_train_batch_size 1 \
+    --per_device_eval_batch_size 1 \
+    --gradient_accumulation_steps 16 \
+    --evaluation_strategy "no" \
+    --save_strategy "steps" \
+    --save_steps 195 \
```

Comment:
195 steps corresponds to one epoch?

Reply:
@regisss 195 steps is greater than 3 epochs for this case, we added this number as a work-around to avoid CPU OOM we see when checkpoint save is triggered at the end of training.  Now we have figured out that we can avoid full model checkpoint (by not adding --save_steps) and instead save only LoRA trained parameters from run_lora_clm.py, so we will remove --save_steps 195 altogether.